apps/general/models.py

```diff
@@ -1,3 +1,230 @@
+    pin_code = models.CharField(max_length=20, null=True, default=None)
+    phone_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="Phone number must be entered in the format: '+999999999'. Up to 15 digits allowed.")
+    phone = models.CharField(validators=[phone_regex], max_length=20, null=True, default=None)
+    email = models.EmailField(max_length=255, null=True)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6, null=True, default=None)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6, null=True, default=None)
+    print_address = models.TextField(null=True, default=None)
+    website = models.URLField(max_length=255, default=None, null=True)
+    facebook_url = models.URLField(max_length=255, default=None, null=True)
+    skype_id = models.CharField(max_length=50, default=None, null=True)
+    twitter_handle = models.CharField(max_length=50, default=None, null=True)
+    linkedin_url = models.URLField(max_length=255, default=None, null=True)
+    pan = models.CharField(max_length=50, default=None, null=True)
+    tan = models.CharField(max_length=50, default=None, null=True)
+    cin = models.CharField(max_length=50, default=None, null=True)
+    gst_tin = models.CharField(max_length=50, default=None, null=True)
+    establishment_code = models.CharField(max_length=50, default=None, null=True)
+    esi_no = models.CharField(max_length=50, default=None, null=True)
+    pf_no = models.CharField(max_length=50, default=None, null=True)
+    authorized_person = models.CharField(max_length=255, default=None, null=True)
+    iec_code = models.CharField(max_length=50, default=None, null=True)
+    eway_username = models.CharField(max_length=100, default=None, null=True)
+    eway_password = models.CharField(max_length=100, default=None, null=True)
+    gstn_username = models.CharField(max_length=100, default=None, null=True)
+    gstn_password = models.CharField(max_length=100, default=None, null=True)
+    VAT_GST_STATUS_CHOICES = (
+        ('Active', 'Active'),
+        ('Inactive', 'Inactive'),
+        ('Pending', 'Pending'),
+    )
+    vat_gst_status = models.CharField(max_length=10, choices=VAT_GST_STATUS_CHOICES, default=None, null=True)
+    GST_TYPE_CHOICES = (
+        ('Goods', 'Goods'),
+        ('Service', 'Service'),
+    )
+    gst_type = models.CharField(max_length=10, choices=GST_TYPE_CHOICES, default=None, null=True)
+    einvoice_approved_only = models.BooleanField(default=False)
+    marketplace_url = models.URLField(max_length=255, default=None, null=True)
+    drug_license_no = models.CharField(max_length=50, default=None, null=True)
+    other_license_1 = models.CharField(max_length=50, default=None, null=True)
+    other_license_2 = models.CharField(max_length=50, default=None, null=True)
+    turnover_less_than_5cr = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
This should satisfy the below command in the table, did u test?

updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP